projects/1.18/assets/malum/malum/lang/zh_cn.json

```
@@ -0,0 +1,692 @@
+  "item.malum.fancy_leggings": "礼服长裤",
+  "item.malum.fancy_tophat": "高顶礼帽",
+  "item.malum.filled_soul_vial": "装有灵魂的灵魂瓶",
+  "item.malum.filled_soulwood_stave": "萦魂灵魂木杖",
+  "item.malum.filled_spirit_jar": "装有灵神的灵神罐",
+  "item.malum.gilded_belt": "镀金腰带",
+  "item.malum.gilded_ring": "镀金戒指",
+  "item.malum.gold_impetus": "金炼金促符",
+  "item.malum.hallowed_gold_ingot": "神圣金锭",
+  "item.malum.hallowed_gold_nugget": "神圣金粒",
+  "item.malum.hallowed_spirit_resonator": "神圣金灵神共振器",
+  "item.malum.hex_ash": "咒灰",
+  "item.malum.holy_sap": "神圣树脂",
+  "item.malum.holy_sapball": "神圣树脂球",
+  "item.malum.holy_syrup": "神圣树脂浆",
+  "item.malum.infernal_spirit": "狱火灵神",
+  "item.malum.iron_impetus": "铁炼金促符",
+  "item.malum.lead_impetus": "铅炼金促符",
+  "item.malum.magebane_belt": "巫灾腰带",
+  "item.malum.necklace_of_the_mystic_mirror": "神秘透镜项链",
+  "item.malum.necklace_of_the_narrow_edge": "精准镰刃项链",
+  "item.malum.ornate_necklace": "华丽项链",
+  "item.malum.ornate_ring": "华丽戒指",
+  "item.malum.poppet": "巫毒人偶",
+  "item.malum.processed_soulstone": "精制灵魂石",
+  "item.malum.ring_of_arcane_reach": "灵神吸引戒指",
+  "item.malum.ring_of_arcane_spoil": "灵神掠夺戒指",
+  "item.malum.ring_of_curative_talent": "疗愈魔能戒指",
+  "item.malum.ring_of_prowess": "经验晶石戒指",
+  "item.malum.ring_of_wicked_intent": "凋灵魔能戒指",
+  "item.malum.runewood_boat": "符文木船",
+  "item.malum.sacred_spirit": "神圣灵神",
+  "item.malum.silver_impetus": "银炼金促符",
+  "item.malum.soul_hunter_boots": "灵魂猎人靴子",
+  "item.malum.soul_hunter_cloak": "灵魂猎人斗篷",
+  "item.malum.soul_hunter_leggings": "灵魂猎人护腿",
+  "item.malum.soul_hunter_robe": "灵魂猎人长袍",
+  "item.malum.soul_stained_steel_axe": "魂染钢斧",
+  "item.malum.soul_stained_steel_boots": "魂染钢靴子",
+  "item.malum.soul_stained_steel_chestplate": "魂染钢胸甲",
+  "item.malum.soul_stained_steel_helmet": "魂染钢头盔",
+  "item.malum.soul_stained_steel_hoe": "魂染钢锄",
+  "item.malum.soul_stained_steel_ingot": "魂染钢锭",
+  "item.malum.soul_stained_steel_leggings": "魂染钢护腿",
+  "item.malum.soul_stained_steel_nugget": "魂染钢粒",
+  "item.malum.soul_stained_steel_pickaxe": "魂染钢镐",
+  "item.malum.soul_stained_steel_scythe": "魂染钢镰刀",
+  "item.malum.soul_stained_steel_shovel": "魂染钢锹",
+  "item.malum.soul_stained_steel_sword": "魂染钢剑",
+  "item.malum.soulstone_cluster": "灵魂石簇",
+  "item.malum.soulwood_boat": "灵魂木船",
+  "item.malum.soulwood_stave": "灵魂木杖",
+  "item.malum.spectral_lens": "灵魂透镜",
+  "item.malum.spirit_fabric": "灵神布匹",
+  "item.malum.spirit_pouch": "灵神布袋",
+  "item.malum.stained_spirit_resonator": "魂染钢灵神共振器",
+  "item.malum.token_of_gratitude": "感恩代币",
+  "item.malum.tyrving": "提尔锋",
+  "item.malum.unholy_sap": "渎神树脂",
+  "item.malum.unholy_sapball": "渎神树脂球",
+  "item.malum.unholy_syrup": "渎神树脂浆",
+  "item.malum.vitric_impetus": "钻石炼金促符",
+  "item.malum.warded_belt": "护卫腰带",
+  "item.malum.wicked_spirit": "邪恶灵神",
+  "itemGroup.malum": "灵灾",
+  "itemGroup.malum_natural_wonders": "灵灾 | 自然奇观",
+  "itemGroup.malum_shaped_stones": "灵灾 | 形制石头",
+  "itemGroup.malum_spirits": "灵灾 | 灵神",

```

```suggestion
  "itemGroup.malum_natural_wonders": "灵灾丨自然奇观",
  "itemGroup.malum_shaped_stones": "灵灾丨形制石头",
  "itemGroup.malum_spirits": "灵灾丨灵神",
```
[TransRules](https://github.com/CFPAOrg/TransRules/blob/master/file/rules.md#07mod-%E5%90%8D%E7%BF%BB%E8%AF%91)